dev-candidates.md

```diff
@@ -1,22 +1,21 @@
 # Workpath Developer Candidate Code Project
 
-In order to help us get a better sense of your approach to coding and ability as a developer, we have designed a small code exercise.  The goal of the exercise is to build a simple browser-based user interface for the publicly available NY Times "Top News" API.  Users should be able to do keyword searches for articles, and to select the section of the newspaper to display articles from.  
+In order to help us get a better sense of your approach to coding and ability as a developer, we have designed a small code exercise.  The goal of the exercise is to build a simple user interface for the publicly available NY Times "Top News" API.  Users should be able to do keyword searches for articles, and to select the section of the newspaper to display articles from.  
 
-Your submission should be a serverless "single-page" solution that can be loaded from the local filesystem.  You may include as many supporting files and assets as you like, but the finished project should be contained in a single zip archive or tarball.  If you decide to use a framework that produces a compiled version of your code (such as React), be sure to include your original source files with your submission.
+Your submission should be a full-stack solution in a language of your preference.  You may include as many supporting files and assets as you like, but the finished project should be contained in a single zip archive or tarball.  If you decide to use a framework that produces a compiled version of your code (such as React), be sure to include your original source files with your submission.
```

Comment:
```suggestion
Your submission should be a full-stack solution in JavaScript, Ruby or Elixir.  You may include as many supporting files and assets as you like, but the finished project should be contained in a single zip archive or tarball.  If you decide to use a framework that produces a compiled version of your code, be sure to include your original source files with your submission.
```

I don't like limiting it, but I feel like we should have some knowledge of the language used so that we can have an educated discussion. 😅

Feel free to ignore if you don't agree.

Reply:
@epogue I agree, but what languages are we going to allow?

Reply:
@epogue could allow Ruby since you know it well.

Reply:
Yep, I suggested JavaScript, Ruby, and Elixir.

Reply:
Including JS and Ruby should make it pretty approachable. Adding Python would probably cover 95%, but I don't know Python that well right now.

I refuse to accept Java. 😅

Reply:
Ok yeah, that works for me too.